dev-manual/api/api-reference-archivematica.rst

```diff
@@ -484,13 +484,45 @@ Response definitions:
 
 ==================   ==========================================================
 
-``message``          "Approval successful."
+``error``            Boolean true or false.
 
-``reingest_uuid``    UUID of the reingested transfer.
+``message``          Success message or reason for the error.
 
 ==================   ==========================================================
 
 
+Usage example::
+
+  curl -X POST \
+  http://my_archivematica_instance.archivematica.org/api/ingest/copy_metadata_files/ \
+  -H 'Authorization: ApiKey [_your_username_]:[_your_ApiKey_]' \
+  -H 'Content-Type: application/x-www-form-urlencoded' \
+  -d 'sip_uuid=a7f1f36c-55c1-4fb7-a898-76d102e847dc&source_paths%5B%5D='''NmRjYTgxNDAtMDg2NC00MzE5LTg2ZDctNTg0ZjZiZTY4N2EzOi9ob21lL2FyY2hpdmVtYXRpY2EvYXJjaGl2ZW1hdGljYS1zYW1wbGVkYXRhL2lzc3VlX3RlbXBsYXRlLm1k''''
```

Comment:
Indentation on the page is not very consistent, but I noticed the `Response examples` below use 4 spaces indentation. Also extra indentation on the `curl` flags might be useful.

```suggestion
    curl -X POST \
        http://my_archivematica_instance.archivematica.org/api/ingest/copy_metadata_files/ \
        -H 'Authorization: ApiKey [_your_username_]:[_your_ApiKey_]' \
        -H 'Content-Type: application/x-www-form-urlencoded' \
        -d 'sip_uuid=a7f1f36c-55c1-4fb7-a898-76d102e847dc&source_paths%5B%5D='''NmRjYTgxNDAtMDg2NC00MzE5LTg2ZDctNTg0ZjZiZTY4N2EzOi9ob21lL2FyY2hpdmVtYXRpY2EvYXJjaGl2ZW1hdGljYS1zYW1wbGVkYXRhL2lzc3VlX3RlbXBsYXRlLm1k''''
```